gapidapk/packageinfo.go

```diff
@@ -46,7 +46,15 @@ func PackageList(ctx context.Context, d adb.Device, includeIcons bool, iconDensi
 		return nil, log.Err(ctx, nil, "Service intent was not found")
 	}
 
-	onlyDebug := d.Root(ctx) == adb.ErrDeviceNotRooted
+	// Can only capture debug application unless Android build is userdebug and adb root works
+	onlyDebug := true
+	userdebugBuild, err := d.UserdebugBuild(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
Shouldn't this be 
```return nil, log.Err(ctx, err, "Starting service")```
as in code blocks below?

Reply:
Yes indeed! fixed.